libs/components/atoms/src/lib/empty-biomarker/empty-biomarker.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="empty-message">{{ emptyMessage }}</div>
```

Comment:
Combine with contact us and change to markdown input (use ngx-markdown)
Class name: `message`